editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2929,6 +2929,51 @@ static void override_button_stylebox(Button *p_button, const Ref<StyleBox> p_sty
 
 void Node3DEditorViewport::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_TRANSLATION_CHANGED: {
+			_update_name();
```

Comment:
The `view_display_menu` should have auto translation disabled since the text is updated here.